config/initializers/highlight_override.rb

```diff
@@ -0,0 +1,31 @@
+# Monkey patch the generated Bindings for the swagger Highlight model
+require_relative '../../app/bindings/api/v0/bindings/new_highlight'
```

Comment:
This require isn't needed because the bindings are autoloaded

Reply:
using class_eval to autoload the NewHighlight binding prior to monkey patching

Reply:
this was changed to a class_eval, so will remove the require_relative